controllers/os-suse-jeos/pkg/generator/templates/cloud-init/suse-jeos.template

```diff
@@ -28,6 +28,7 @@ write_files:
 runcmd:
 - "until zypper -q install -y docker wget socat jq nfs-client; [ $? -ne 7 ]; do sleep 1; done"
 - systemctl daemon-reload
+- ip link set dev eth0 mtu 1460
```

Comment:
do this survive reboots? how can this change be permanent @rossella?

adding add/change MTU='1460' in /etc/sysconfig/network/ifcfg-eth0?

Reply:
> do this survive reboots?

Gardener never reboot the VMs, but of course it is possible to happen. 
I've added a command to set the MTU in `/etc/sysconfig/network/ifcfg-eth0` file.